core/src/main/java/com/klaviyo/coresdk/Klaviyo.kt

```diff
@@ -6,13 +6,13 @@ import com.klaviyo.coresdk.networking.requests.KlaviyoRequest
 import com.klaviyo.coresdk.networking.requests.TrackRequest
 
 object Klaviyo {
-    fun track(event: KlaviyoEvent, customerProperties: Map<String, String>, properties: Map<String, String>? = null) {
+    fun track(event: KlaviyoEvent, customerProperties: MutableMap<String, String>, properties: Map<String, String>? = null) {
```

Comment:
Needs to be a `MutableMap` now to support the anonymous IDs that we will tack on